docs/api/event-brokers.rst

```diff
@@ -8,9 +8,23 @@ Event Brokers
 
 .. edit-link::
 
-Rasa Core allows you to stream events to a message broker. The event broker
-emits events into the event queue. It becomes part of the ``TrackerStore``
-which you use when starting an ``Agent`` or launch ``rasa.core.run``.
+When running a Rasa Open Source assistant in production, you might want to
+connect your assistant to other services so that they can analyze the data
+coming in from your conversations. For example, you could `connect your
+live assistant to Rasa X <https://rasa.com/docs/rasa-x/installation-and-setup/existing-deployment/>`_
+to review and annotate conversations, or forward messages to an external analytics
+service. 
+
+To do so, Rasa allows you to stream events using an event broker. The event broker
+publishes messages to a message streaming service, also known as a message broker,
+to forward Rasa :ref:`events` from the Rasa server to other services.
+
+.. contents::
+   :local:
+   :depth: 1
+
+Format
+------
 
 All events are streamed to the broker as serialised dictionaries every time
 the tracker updates it state. An example event emitted from the ``default``
```

Comment:
go ahead an update `it state` to `its state`?

Reply:
good catch!